### ISSUE TITLE
mkdir: cannot create directory - setup-hadoop.sh

### ISSUE THREAD

**Issue by jhajek** [SAT-Hadoop/SetupScripts#4]
qemu: Stopping namenodes on [master]
    qemu: master: Warning: Permanently added 'master,10.0.2.15' (ECDSA) to the list of known hosts.
    qemu: master: mkdir: cannot create directory ‘/mnt/vol-01’: Permission denied
    qemu: master: chown: cannot access ‘/mnt/vol-01/hadooplogs/’: No such file or directory
    qemu: master: no namenode to stop
    qemu: master: Warning: Permanently added 'master,10.0.2.15' (ECDSA) to the list of known hosts.
    qemu: master: mkdir: cannot create directory ‘/mnt/vol-01’: Permission denied
    qemu: master: chown: cannot access ‘/mnt/vol-01/hadooplogs/’: No such file or directory
    qemu: master: no datanode to stop
    qemu: Stopping secondary namenodes [0.0.0.0]
    qemu: 0.0.0.0: Warning: Permanently added '0.0.0.0' (ECDSA) to the list of known hosts.
    qemu: 0.0.0.0: mkdir: cannot create directory ‘/mnt/vol-01’: Permission denied
    qemu: 0.0.0.0: chown: cannot access ‘/mnt/vol-01/hadooplogs/’: No such file or directory
    qemu: 0.0.0.0: no secondarynamenode to stop
    qemu: stopping yarn daemons
    qemu: mkdir: cannot create directory ‘/mnt/vol-01’: Permission denied
    qemu: chown: cannot access ‘/mnt/vol-01/hadooplogs/’: No such file or directory
    qemu: no resourcemanager to stop
    qemu: master: Warning: Permanently added 'master,10.0.2.15' (ECDSA) to the list of known hosts.
    qemu: master: mkdir: cannot create directory ‘/mnt/vol-01’: Permission denied
    qemu: master: chown: cannot access ‘/mnt/vol-01/hadooplogs/’: No such file or directory
    qemu: master: no nodemanager to stop
    qemu: mkdir: cannot create directory ‘/mnt/vol-01’: Permission denied
    qemu: chown: cannot access ‘/mnt/vol-01/hadooplogs/’: No such file or directory
    qemu: no proxyserver to stop

**Comment by saipramod** [comment]
We need have an EBS attached at the time of instance launch and the volume would be mounted to /vol-01/

**Comment by jhajek** [comment]
Ok that makes sense - we need to detail the launch script and see if we can
arrange this mounting...  also what IAM profile are we using?

On Sun, Jun 14, 2015 at 3:38 PM, Sai Pramod Upadhyayula <
notifications@github.com> wrote:

> We need have an EBS attached at the time of instance launch and the volume
> would be mounted to /vol-01/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SAT-Hadoop/SetupScripts/issues/4#issuecomment-111874794>
> .
>



-- 
Jeremy Hajek
Industry Associate Professor of Information Technology and Management
ext: 630-682-6075 (2-6075)
Main: 312-567-5290 (7-5290)
cell: (630) 296-4012
skype: jeremy.hajek

**Comment by saipramod** [comment]
None. As of now we are entirely on Eucalyptus. No Amazon.

**Comment by jhajek** [comment]
Ok.  Eucalyptus has iam support too.  So just the build system needs euca credentials.  Or we need to move the mounting and esb creation to the launch script (euca-run-instances) as opposed to the build.   What is this mount point on ebs doing? Logs?

-----Original Message-----

**Comment by saipramod** [comment]
Yes. we need to move it to launch script. I updated the instructions of the
setup scripts repo , for the backend worker. Please let me know if it s not
clear. Yes, ebs is for logs.

**Comment by jhajek** [comment]
If I comment out the last two lines (the stop-all.sh) and the Hadoop master --force and move those to the euca-run-instances script then I think I solve our chicken and egg problem of not having the EBS mounts at build time

**Comment by saipramod** [comment]
Sure thing.
